CriticalMass/FormatDisplay.swift

```diff
@@ -29,8 +29,11 @@ private extension DateComponents {
 }
 
 enum FormatDisplay {
+    // inject current date to be able to keep the same time difference between tweet date and current date
+    static var currentDate = Date()
```

Comment:
Not a fan of having a static var here and would prefer injecting the date as an argument or initialiser but nothing which we can easily do with the current architecture here but that should be okay for no

Reply:
Yep, maybe we could refactor it in the future.